src/display/api.js

```diff
@@ -1199,9 +1200,11 @@ class PDFDocumentProxy {
  *   The default value is `AnnotationMode.ENABLE`.
  * @property {Array<any>} [transform] - Additional transform, applied just
  *   before viewport transform.
- * @property {Object} [canvasFactory] - The factory instance that will be used
- *   when creating canvases. The default value is {new DOMCanvasFactory()}.
- * @property {Object | string} [background] - Background to use for the canvas.
+ * @property {BaseCanvasFactory} [canvasFactory] - The factory instance that
+ *   will be used when creating canvases. The default value is
+ *   {new DOMCanvasFactory()}.
```

Comment:
That parameter is first of all deprecated, which means that there's no reason to update it. Secondly, the user may provide a custom factory that's *not* actually `instanceof BaseCanvasFactory` which means that this change wasn't entirely accurate; please just revert this change.